### PR TITLE
Update Gopkg.toml manually

### PR DESCRIPTION
**Pull request by JadeRedworth**

### CODE REVIEW

**Comment by msgodf** [comment]
Looks good - to clarify, apparently this is the way to update a dep version: https://github.com/golang/dep/blob/master/docs/FAQ.md#how-do-i-change-the-version-of-a-dependency (we didn't realise this when updating fn_go).

**Comment by msgodf** [comment]
Looks good - to clarify, apparently this is the way to update a dep version: https://github.com/golang/dep/blob/master/docs/FAQ.md#how-do-i-change-the-version-of-a-dependency (we didn't realise this when updating fn_go).

File: Gopkg.toml [hunk 1]
```diff
@@ -34,7 +34,7 @@
 
 [[constraint]]
   name = "github.com/fnproject/fn_go"
-  version = "0.2.7"
+  version = "0.2.8"
```
Comment (zootalures) [inline Gopkg.toml]: You should run a dep-ensure locally to update the lock file and check that in  (otherwise dep ensure --vendor-only craps itself)

Comment (JadeRedworth) [reply Gopkg.toml]: Instead of changing it manually?

**Pull request merged**